dashboard/run_data_collection.py

```diff
@@ -274,6 +327,36 @@ def _parse_quota(self, lines: Iterable[str]):
 
         return entries
 
+    def _parse_user_chours_usage(self, lines: Iterable[str], users: Iterable[str]):
+        """Get daily core hours usage for a list of users from cmd output"""
+        entries = []
+        # if none of the user had usage today, lines=['']
+        # Then we just set core_hour_used to 0 for all users
+        if lines == [""]:
+            for user in users:
+                entries.append(UserChEntry(date.today(), user, 0))
+        # if some of the users had usages today
+        #                                                        used
+        # ['maui       nesi00213    jpa198 James Paterson+        1        0 ',
+        #   maui       nesi00213     tdn27 Andrei Nguyen +      175        0']
+        else:
+            used_users = set()
+            for ix, line in enumerate(lines):
+                line = line.split()
+                used_users.add(line[2])
+                try:
+                    entries.append(UserChEntry(date.today(), line[2], line[-2]))
+                except ValueError:
+                    print(
+                        "Failed to convert squeue line \n{}\n to "
+                        "a valid SQueueEntry. Ignored!".format(line)
```

Comment:
The error message needs a little bit of updating.

Reply:
All updated